### PR TITLE
adding workflows

### PR DESCRIPTION
**Pull request by jengelhard**
Added workflows.
Took out black for the time being, until we add testing. 
Changed python version to 3.8 because greenlet was giving issues.

### CODE REVIEW

File: dev-requirements.in [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # lint
 black
```
Comment (NJABULO-NDLOVU) [inline dev-requirements.in:2]: `black <= 18`

File: Justfile [hunk 2]
```diff
@@ -23,12 +23,10 @@ requirements:
 
 # Setup requirements
 setup:
-	@just requirements
```
Comment (NJABULO-NDLOVU) [inline Justfile:26]: keep this line

File: .github/workflows/package.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    name: PyPI Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v2
+
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```
Comment (NJABULO-NDLOVU) [inline .github/workflows/package.yml:16]: 3.10

File: Justfile [hunk 4]
```diff
@@ -19,7 +19,7 @@ install-pip-tools:
 # Create requirements.txt file
 requirements:
 	@just install-pip-tools
-	pip-compile
```
Comment (NJABULO-NDLOVU) [inline Justfile:22]: `pip install pipenv ` before running pipenv lock

Comment (NJABULO-NDLOVU) [reply Justfile:22]: then comment that it's only there temporarily

File: README.md [hunk 5]
```diff
@@ -1,70 +1,41 @@
-# celery-sqlalchemy-scheduler
 
-A Scheduler Based Sqlalchemy for Celery.
+[![code](https://github.com/aruba-uxi/celery-sqlalchemy-scheduler/actions/workflows/lint-test-code.yaml/badge.svg)](https://github.com/aruba-uxi/celery-sqlalchemy-scheduler/actions/workflows/lint-test-code.yaml)
 
-> NOTE: At first I developed this project for flask with celery to change scheduler from database, like [django-celery-beat](https://github.com/celery/django-celery-beat) for django. And now I maybe haven't time to develop for new feature. No new feature develop plan for it. Just fix bugs. If someone found some bugs, welcome to issue or PR. Thank you for your attention.
+[![Python Version](https://img.shields.io/badge/python-3.10-blue?logo=Python&logoColor=yellow)](https://docs.python.org/3.10/)
```
Comment (evanstjabadi) [inline README.md:4]: Put 3.8 to correspond with the version

**Review by evanstjabadi — CHANGES_REQUESTED** [review]

**Review by evanstjabadi — APPROVED** [review]
LGMT

**Pull request merged**